src/Drivers/Translation.php

```diff
@@ -89,12 +89,19 @@ public function getGoogleTranslate($language,$token){
      * @param $language
      */
     public function translateLanguage($language){
+        //No need to translate e.g. English to English
+        if ($language === $this->sourceLanguage) {
+            return;
+        }
+
         $translations = $this->getSourceLanguageTranslationsWith($language);
 
         foreach ($translations as $type => $groups) {
             foreach ($groups as $group => $translations) {
                 foreach ($translations as $key => $value) {
-                    $sourceLanguageValue = $value[$this->sourceLanguage];
+                    //Value will be empty if it's found in the app source code but not in the source language files
```

Comment:
The auto-translates first calls ` $this->saveMissingTranslations($language);`  which is command `php artisan translation:sync-missing-translation-keys`
This is to ensure that all keys in source code will be in the source language files.

But it is still a good fail safe